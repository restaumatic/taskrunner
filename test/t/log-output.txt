# check logs/mytask.log
taskrunner -n mytask sh -c '
  echo "some standard output"
  echo "some standard error" >&2
  echo "more standard output"
'
