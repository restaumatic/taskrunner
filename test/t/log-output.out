-- output:
[mytask] stdout | some standard output
[mytask] stdout | more standard output
[mytask] stderr | some standard error
-- logs/mytask.log:
stdout | some standard output
stdout | more standard output
stderr | some standard error
